README.md

```diff
@@ -21,13 +21,14 @@
   - [Install](#install)
   - [Usage](#usage)
   - [Run tests](#run-tests)
-  - [Deployment](#deployment)
 - [👥 Authors](#authors)
 - [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
 - [📝 License](#license)
+- [Deployment](#deployment)
```

Comment:
- [ ] In this milestone, you are supposed to make your portfolio accessible to the public by deploying it with the GitHub page. But in the PR submitted, there is no presence of the application deployed link. **Kindly update the README of your repository to include a link to the online version**. You can read more about how to deploy your application with the GitHub page [here](https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site) :smile: :wave: :+1: 

- [ ] Additionally, please consider summarizing the changes you've made in the pull request (PR) description in a more detailed fashion. To give the reviewer a clear understanding of what you've accomplished and how it aligns with the project goals, please provide a detailed summary of the changes you've made in your own words. This will help ensure that your PR is reviewed thoroughly and efficiently. :wave: :+1: :100:

Reply:
I have changed the README.md file now with the Deployment link added to it but i cant find out from where i can change the PR description. Please guide me on it

![image](https://github.com/siddghosh108/Portfolio-Desktop-Version/assets/131663359/3d848e18-de8a-4802-a198-6a983a5e62d2)